bin/p2-rctl/main.go

```diff
@@ -335,7 +335,7 @@ LOOP:
 }
 
 func (r RCtl) ScheduleUpdate(oldID, newID string, want, need int) {
-	err := r.rls.Put(roll_fields.Update{
+	_, err := r.rls.CreateRollingUpdateFromExistingRCs(roll_fields.Update{
```

Comment:
Future request: it'd be great to get all three creation methods into `p2-rctl` as part of the CLI.